client/src/ingredients/examples.js

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+
+function RenderSources({ sources }) {
+  return (
+    <>
+      {sources.html && <h4>HTML</h4>}
+      {/* <code> inside a <pre>?? Is that necessary? */}
```

Comment:
@wbamberg Can you address this or delete the code comment?

Reply:
This seems to be the desired practice for code blocks: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/code#Notes